db/db_test_util.h

```diff
@@ -967,23 +967,33 @@ class DBTestBase : public testing::Test {
 
   DBImpl* dbfull() { return static_cast_with_check<DBImpl>(db_); }
 
-  void CreateColumnFamilies(const std::vector<std::string>& cfs,
-                            const Options& options);
+  void CreateColumnFamilies(
+      const std::vector<std::string>& cfs, const Options& options,
+      std::unordered_map<std::string, std::shared_ptr<WriteBufferManager>>
```

Comment:
This is not the intended use case of this function. If you have different options for each cf, use `ReopenWithColumnFamilies(const std::vector<std::string>& cfs, const std::vector<Options>& options);` instead.

Reply:
fixed